arrays/arrays.c

```diff
@@ -146,11 +146,23 @@ void arr_remove(Array *arr, char *element) {
 
   // Search for the first occurence of the element and remove it.
   // Don't forget to free its memory!
-
+  int index = 0;
+  for(int i = 0; i < arr->count; i++)
```

Comment:
As it is now, this for loop will keep going after finding the first occurrence of the element. Also take into consideration what should happen in the element isn't found at all.